docs/setup/pick-right-solution.md

```diff
@@ -61,6 +61,8 @@ a Kubernetes cluster from scratch.
 
 * [Kubermatic](https://www.loodse.com) provides managed Kubernetes clusters for various public clouds, including AWS and Digital Ocean, as well as on-premises with OpenStack integration.
 
+* [Pivotal Container Services](https://pivotal.io/platform/pivotal-container-service) provides enterprise-grade Kubernetes for both on-premises and public clouds.  PKS enables on-demand provisioning of Kubernetes clusters, multi-tenancy and fully automated day-2 operations.
```

Comment:
Kind of funny that they use PKS as their acronym even though they are called Pivotal Container Services.  I guess when they want you to think Container service, they want you to think something that starts with a K :-)